src/components/TheToolPanel.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <div class="btn-toggle-box ma-0 pa-0">
     <v-btn-toggle class="flex-column">
+      <tool-container :id="id" icon="fa-arrows-alt" toolname="move" :popout="false" />
```

Comment:
Can we use a different icon here than the arrows. This tool will also be used to select entities on the canvas so you won't necessarily be moving a shape all the time. It needs to be a select icon

Reply:
I think Rites is creating the icon atm so will leave this at that